tools/cluster_mgr.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import math
+import redis
+import subprocess
+import time
+
+'''
+To install: pip install -r requirements.txt
+'''
+
+
+class Node:
+    def __init__(self, port, admin_port):
+        self.port = port
+        self.admin_port = admin_port
+
+    id = ''
+
+
+def start_node(node, threads):
+    f = open(f'/tmp/dfly.cluster.node.{node.port}.log', 'w')
+    print(f'- Log file for node {node.port}: {f.name}')
+    subprocess.Popen(['../build-dbg/dragonfly', f'--port={node.port}',
+                      f'--admin_port={node.admin_port}', '--cluster_mode=yes', f'--proactor_threads={threads}',
+                      '--dbfilename=', f'--logtostderr'], stderr=f)
+
+
+def send_command(node, command):
+    for i in range(0, 5):
+        try:
+            client = redis.Redis(decode_responses=True,
+                                 host="localhost", port=node.admin_port)
+            result = client.execute_command(*command)
+            client.close()
+            return result
+        except:
+            time.sleep(0.1)
+
+    print(
+        f'Unable to connect to localhost:{node.admin_port} after 5 attempts!')
+
+
+def update_id(node):
+    id = send_command(node, ['dflycluster', 'myid'])
+    node.id = id
+    print(f'- ID for {node.port}: {id}')
+
+
+def build_config(masters, replicas):
+    slots_per_node = 16384 / len(masters)
```

Comment:
This will be a float in python3, // is strict division

Reply:
I actually want this to be a floating point, I rely on this in the `floor()` operations below :)

Reply:
Oh, didn't see them below. Doing it with floats feels very fragile. I guess all you need is to assign them equal ranges plus the remainder for the last node